RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -1243,16 +1243,15 @@ namespace lst {
     float sdIn_alpha_max = __H2F(segmentsInGPU.dPhiChangeMaxs[innerSegmentIndex]);
     float sdOut_alpha = sdIn_alpha;
 
-    float sdOut_alphaOut = lst::phi_mpi_pi(acc,
-                                           lst::phi(acc,
-                                                    mdsInGPU.anchorX[fourthMDIndex] - mdsInGPU.anchorX[thirdMDIndex],
-                                                    mdsInGPU.anchorY[fourthMDIndex] - mdsInGPU.anchorY[thirdMDIndex]) -
-                                               mdsInGPU.anchorPhi[fourthMDIndex]);
```

Comment:
indeed, this looks like a part from the barrel mixed up with the endcap logic below.
Although